shenyu-agent/shenyu-agent-core/src/main/java/org/apache/shenyu/agent/core/utils/ShenyuAgentConfigUtils.java

```diff
@@ -55,8 +55,7 @@ private ShenyuAgentConfigUtils() {
     public static Map<String, AgentPluginConfig> getPluginConfigMap() {
         return getConfig().getPlugins().values().stream()
                 .map(Map::entrySet).flatMap(Set::stream)
-                .collect(Collectors.toMap(Entry::getKey, entry -> GsonUtils.getInstance()
-                        .fromJson(GsonUtils.getInstance().toJson(entry.getValue()), AgentPluginConfig.class)));
+                .collect(Collectors.toMap(Entry::getKey, Entry::getValue));
```

Comment:
not modfiy this

Reply:
get value is map， not convert to AgentPluginConfig， so mybe used jsonUtils from json to AgentPluginConfig

Reply:
you can see final data is map not AgentPluginConfig